src/FluentMigrator.Abstractions/Infrastructure/ErrorMessages.de-DE.resx

```diff
@@ -96,4 +201,13 @@
   <data name="ExpressionTableNameMissingWithHints" xml:space="preserve">
     <value>Der Tabellenname wurde nicht angegeben. Stellen Sie sicher, dass sie die "OnTable"-Erweiterungsmethode aufgerufen haben. Der Aufruf sollte wie folgt aussehen: Delete.ForeignKey(KeyName).OnTable(TableName)</value>
   </data>
+  <data name="IndexIncludeColumnNameCannotBeNullOrEmpty" xml:space="preserve">
+    <value>Der Name der Index-Include-Spalte darf nicht null oder leer sein.</value>
+  </data>
+  <data name="ForeignKeyCannotHaveColumnNameBeNullOrEmpty" xml:space="preserve">
+    <value>Der Fremdschlüssel-Name darf nicht null oder leer sein.</value>
+  </data>
+  <data name="ForeignKeysPrimaryKeyCannotHaveColumnNameBeNullOrEmpty" xml:space="preserve">
+    <value>Der primäre der Fremdschlüssel-Name darf nicht null oder leer sein.</value>
```

Comment:
I guess I would translate it as follows:

Der Spaltenname des Primärschlussels des Fremdschlüssels darf nicht null oder leer sein.

The wording "foreign key's primary key" sounds strange. What is meant exactly?

Reply:
Thanks, I incorporated the additional suggestions we had over direct message.